### PR TITLE
Set up a 'Hello world' project as one app with Webpack

### PR DESCRIPTION
**Pull request by LokurasRlz**
# In this branch, I have worked on the following things: 

- Created a Development and feature branch according to GitHub workflow rules.
- Added React and Redux successfully to the application.
- Created API endpoint for greetings.
- Added a feature to display predefined random greetings every time a user refreshes the page.
- Integrated React, Redux, and Rails successfully.

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import Greeting from './Greeting';
+
+function App() {
+  return (
+    <main>
+      <div className="container">
+        <Greeting />
+      </div>
+    </main>
+  );
```
Comment (t-yanick) [inline app/javascript/components/App.js:12]: - [x] Great job with the project. As per project requirements, the `App` component is supposed to have a route that leads to the `Greeting` component. Kindly check and get that added.
image
`Create the Greeting component that will display a greeting. Set it up as a route in your App component.
`

File: README.md [hunk 2]
```diff
@@ -1 +1,185 @@
-# hello-rails-react
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```
Comment (t-yanick) [inline README.md]: - [x] Kindly remove this logo code. You can use your personal logo. Do this for all your future projects.

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @LokurasRlz , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Linters all checked
:heavy_check_mark: Descriptive PR

There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Having more commits. Commit every new piece of code.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @LokurasRlz , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️ No linter errors
✔️ Gitflow was correctly used


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**